README.md

```diff
@@ -1 +1,21 @@
 # Requests
+
+# Draft of the Rules
+
+- Anyone can join @morganey-lang organization as a Member
+
+- If someone have an Open Sourced Morganey related project, they can join the organization as a Project Leader by transfering the ownership of the project to @morganey-lang. What that means:
+
+    - The Project Leader preserves all of the legal rights on the transfered project. From the technical point of view I will make sure that after the transfering the ownership the Project Leader has enough control over their repo.
+
+    - The Project Leader can transfer their project back. From the technical point of view I will help to do that.
+
+    - The Project Leader continues to decide on how to organize the development process of the transfered repo.
+
+    - The Project Leader can transfer the project to @morganey-lang completely which includes (but not limited to) transfering all of the legal rights on the project, ability to transfer the ownership of the project back, ability to organize the development process of the project. (This is required if the Project Leader cannot work on the project anymore and don't want the project to die. They simply donate the project to the organization).
+
+    - Transfering the project gives
+
+        - featuring the project on the morganey web-site (we probably gonna have one)
+        - more contributors from the members and the neighbor projects
+        - ???
```

Comment:
Looks Good to me I like the idea of when a project leader submits a repo they have the option to take it back at any point if they want to unlike some organizations which have complex CLA agreements. Thanks a lot for allowing me to become a part of the @morganey-lang/intellij-idea-plugin-team I think Morganey already has a great community and it will continue to grow as time progresses